src/hptlink.c

```diff
@@ -772,8 +772,13 @@ void linkArea(s_area * area)
 void usage(void)
 {
     printf(
-        "Usage: hptlink [options] [areaname ...]\n" "Options:  -t\t- build reply TREE\n" "\t  -s\t- do not use Subject\n" "\t  -a\t- search in all messages (for singlethread only)\n" "\t  -r\t- do not use REPLY:/MSGID:\n"
-                                                                                                                                                                                       "\t  -n\t- link with 'new' messages only ('new' from last linked + 1)\n");
+        "Usage: hptlink [-c config] [options] [areaname ...]\n"
```

Comment:
"-c" is just an option, isn't it? So it is better to have
"Usage: hptlink [options] [areaname ...]\n"
and to add "-c" to the options list.

Reply:
I made it similar to hpt help text. Do you want me to do this change or keep it similar to other tools?

Reply:
Hpt help is very strange: "toss", "scan" etc are rather commands and option names begin with "-". So I think it is better to consider "-c" an option. And let us think how it's better to formulate the hpt help.

Reply:
Sounds good. I'll move it to options for both cases.